worlds/kh1/Regions.py

```diff
@@ -0,0 +1,550 @@
+from typing import Dict, List, NamedTuple, Optional
+
+from BaseClasses import MultiWorld, Region, Entrance
+from .Locations import KH1Location, location_table
+
+
+class KH1RegionData(NamedTuple):
+    locations: Optional[List[str]]
+    region_exits: Optional[List[str]]
+
+
+def create_regions(multiworld: MultiWorld, player: int, options):
+    regions: Dict[str, KH1RegionData] = {
+        "Menu":             KH1RegionData(None, ["Awakening", "Levels"]),
+        "Awakening":        KH1RegionData([],   ["Destiny Islands"]),
+        "Destiny Islands":  KH1RegionData([],   ["Traverse Town"]),
+        "Traverse Town":    KH1RegionData([],   ["World Map"]),
+        "Wonderland":       KH1RegionData([],   []),
+        "Olympus Coliseum": KH1RegionData([],   []),
+        "Deep Jungle":      KH1RegionData([],   []),
+        "Agrabah":          KH1RegionData([],   []),
+        "Monstro":          KH1RegionData([],   []),
+        "Atlantica":        KH1RegionData([],   []),
+        "Halloween Town":   KH1RegionData([],   []),
+        "Neverland":        KH1RegionData([],   []),
+        "Hollow Bastion":   KH1RegionData([],   []),
+        "End of the World": KH1RegionData([],   []),
+        "100 Acre Wood":    KH1RegionData([],   []),
+        "Levels":           KH1RegionData([],   []),
+        "World Map":        KH1RegionData([],   ["Wonderland", "Olympus Coliseum", "Deep Jungle",
+                                         "Agrabah", "Monstro", "Atlantica",
+                                         "Halloween Town", "Neverland", "Hollow Bastion",
+                                         "End of the World", "100 Acre Wood"])
+    }
+
+    # Set up locations
+   #regions["Destiny Islands"].locations.append("Destiny Islands Chest"), missable
+    regions["Traverse Town"].locations.append("Traverse Town 1st District Candle Puzzle Chest"),
+    regions["Traverse Town"].locations.append("Traverse Town 1st District Accessory Shop Roof Chest"),
+    regions["Traverse Town"].locations.append("Traverse Town 2nd District Boots and Shoes Awning Chest"),
+    regions["Traverse Town"].locations.append("Traverse Town 2nd District Rooftop Chest"),
+    regions["Traverse Town"].locations.append("Traverse Town 2nd District Gizmo Shop Facade Chest"),
```

Comment:
What are all these commas on the ends of the lines?
It's creating a bunch of tuples of `(None,)` just to throw them away?

Reply:
Good catch, that's completely unnecessary.  Needed to clean up this script anyway, so I removed the commas and just reformatted `Regions.py`, should make it a bit more organized.